From 0d89adac03df523c33861f6d66fa9917ed975ed8 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Thu, 9 Mar 2017 16:15:14 +0200
Subject: [PATCH 0847/1345] fix: dts: cp110: Update CP-110 system controller
 clocks

commit  02338c31aa7ee4855255e63dda244f3c08eaf862 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Add MMC clock to the list of system controller core clocks.

Change-Id: I03463ad66f9e1f38b6e1ab57dce3ec047da645cb
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37360
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 599b725..62a82ea 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -25,7 +25,7 @@ cpm_syscon0: system-controller@440000 {
 	#clock-cells = <2>;
 	core-clock-output-names =
 		"cpm-apll", "cpm-ppv2-core", "cpm-eip",
-		"cpm-core", "cpm-nand-core";
+		"cpm-core", "cpm-nand-core", "cpm-emmc";
 	gate-clock-output-names =
 		"cpm-audio", "cpm-communit", "cpm-nand",
 		"cpm-ppv2", "cpm-sdio", "cpm-mg-domain",
-- 
1.7.9.5

